proposals/0013-wave-size-range.md

```diff
@@ -125,16 +132,20 @@ These are where new or slightly altered errors are produced:
 
 Validation should confirm:
 
-* Each element in that is a power-of-two integer between 4 and 128 inclusive.
-  The third parameter may also be zero.
-* The minimum wave size is less than the maximum wave size.
-* The preferred wave size is zero or lies between the minimum and maximum.
-* Where validator version is less than 1.8, fail on shaders that use
- `RangedWaveSize`.
-* Validation should fail in an entry has both `RangedWaveSize` and `WaveSize`
- metadata.
-* Where validator version is greater than or equal to 1.8, fail when The tuple
- that the wave size range tag points to does not have exactly three elements.
+* The metadata `kDxilRangedWaveSizeTag` points to a tuple of exactly three
+ elements.
+* Each element of that tuple is a power-of-two integer between 4 and 128
+ inclusive or zero.
+* However, the first element (minimum wave size) is not zero.
+* The first element (minimum wave size) is less than the second
+ (maximum wave size).
+* The third element (preferred wave size) is greater or equal to the first
+ element (minimum wave size) and less than or equal to the second element
+ (maximum wave size) or zero.
+* Validators with versions less than 1.8 fail on shaders that use
+ `kDxilRangedWaveSizeTag`.
```

Comment:
I have been told that RangedWaveSize tag should fail validation with dxil validator version < 1.8, and I have also been told that it should fail for shader models < 6.8. Is it one or the other, or both? @tex3d

Reply:
It's specifically for shader model < 6.8.  1.8 is the first validator version that accepts shader model 6.8, so prior versions will fail when they encounter the unrecognized tag, and the newer version will not allow you to set a validator version < 1.8 if the shader model is 6.8.